AK/Time.h

```diff
@@ -143,6 +144,36 @@ constexpr i64 seconds_since_epoch_to_year(i64 seconds)
 }
 #endif
 
+template<typename T>
+constexpr Tuple<T, u8, u8> days_to_time(T days)
+{
+    // Adapted from https://howardhinnant.github.io/date_algorithms.html#civil_from_days
+    static_assert(NumericLimits<T>::digits() >= 20, "This algorithm has not been ported to a 16 bit signed integer");
+
+    constexpr u32 average_days_per_year = 365;
```

Comment:
365 is not the average number of days per year, use a different name (or don't have a named constant for this one as 365 looks fairly unsurprising as literal in date math code)